scribejava-core/src/main/java/com/github/scribejava/core/model/OAuth2ErrorResponse.java

```diff
@@ -0,0 +1,36 @@
+package com.github.scribejava.core.model;
+
+import java.net.URI;
+
+// https://tools.ietf.org/html/rfc6749#section-5.2
+public class OAuth2ErrorResponse {
```

Comment:
Do we need this class and OAuthRequestException simultaneously?
Looks like we can have just one class

OAuth2ErrorResponse extends OAuthException
representing "5.2.  Error Response" from https://tools.ietf.org/html/rfc6749#section-5.2